toolset/benchmark/benchmarker.py

```diff
@@ -538,6 +539,14 @@ def exit_with_code(code):
           """)
```

Comment:
Could this PR also include a restart to cassandra here? That way we could address all the database issues with one PR